### ISSUE TITLE
As a user, I want to filter search results by publication year or range of years so that I focus on works from a particular time period.

### ISSUE THREAD

**Issue by thatbudakguy** [Princeton-CDH/ppa-django#30]
## Notes for testing
- placeholder text should display max/min values based on what's in the database
- should be able to specify just a min (everything after date), just a max (everything before), or both
- use same date in both to get items from a single year
- entering a min greater than max should give an error message
- entering a date outside the max/min in the db should give an error message

**Comment by meagan1019** [comment]
Searched "(iamb OR trochee) AND count" returned 200 works. 1559 and 1922 were pre-populated, which are the correct dates! Filtered with start date of 1891 (no end date), works dropped to 112. Filtered with end date of 1820, works dropped to 5. Search 1820-1820, returned the single work published that year. Received error message when I tried to filter outside of parameters (entered 1500 and 1930). However, the thumbnails and text results with italicized search terms did not display when filtering by year (in any iteration I tested). Once I entered anything in the publication year fields (min and/or max) the results no longer displayed the actual results, just the work bibliography.
